processing/src/main/java/io/druid/query/extraction/StringFormatExtractionFn.java

```diff
@@ -107,7 +107,7 @@ public String apply(@Nullable String value)
         value = "";
       }
     }
-    return Strings.emptyToNull(StringUtils.format(format, value));
+    return io.druid.common.config.NullHandling.emptyToNullIfNeeded(StringUtils.format(format, value));
```

Comment:
`NullHandling` could be imported?

Reply:
this class already has an enum with same name.